README.md

```diff
@@ -96,6 +96,12 @@ yarn install && \
 yarn build
 ```
 
+#### Load submodules 
```

Comment:
This instruction is a little disjointed, and I'd prefer to see users setting this up during the initial clone like [here](https://github.com/input-output-hk/cardano-graphql#getting-started). I propose we add a _Clone_ section above _Install and Build_.

Reply:
960a822e33a034c6050054af097131d56ba6002f